src/app.module.ts

```diff
@@ -2,9 +2,9 @@ import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { DiscordModule } from './discord/discord.module';
-
+import { ConfigModule } from '@nestjs/config';
 @Module({
-  imports: [DiscordModule],
+  imports: [ConfigModule.forRoot(), DiscordModule],
```

Comment:
In case we need ConfigModule in other modules as well theres the posibility to import it as a global Module by giving the options-object with the following configuration
```ts
{
  isGlobal: true,
}
``` 
to the `forRoot` function. 
In this case when we're just having a single module this is fine I guess, but as the application grows we can save this import in any others module then and just have to do it once.